zookeeper/Makefile

```diff
@@ -41,6 +42,9 @@ KEYS:
 	@wget -q https://dist.apache.org/repos/dist/release/zookeeper/KEYS
 	gpg --import KEYS
 
+zkauth:
+	cp -v ../../zookeeper-authorizers/build/libs/zookeeper-authorizers.jar .
```

Comment:
nit - at some point i think we need to think about using submodules, so that we do not refer to path outside of git repo.